pages/status.tsx

```diff
@@ -112,15 +112,26 @@ const Status: FC = () => {
                 goBackStyle="primary"
                 checkAgainText={t('check-again')}
               >
-                <p className="mb-6 text-2xl">
-                  {`${t('status-is')} `}
-                  <strong id="response-status">
-                    {t(`status.${checkStatusReponse.status}`, {
-                      defaultValue: checkStatusReponse.status,
-                    })}
-                  </strong>
-                  .
-                </p>
+                <div>
+                  <p className="mb-6 text-2xl">
+                    {`${t('status-is')} `}
+                    <strong id="response-status">
+                      {t(`status.${checkStatusReponse.status}.label`, {
+                        defaultValue: checkStatusReponse.status,
+                      })}
+                    </strong>
+                    .
+                  </p>
+                  <p>{t(`status.${checkStatusReponse.status}.description`)}</p>
+                  {/* We only want to show the tracking # if the status is issued and shipped via Canada Post or Fedex. Currently, some records have a manifest # but are not one of these codes (ie. 99) */}
+                  {checkStatusReponse.manifestNumber &&
+                    ['3', '4'].indexOf(checkStatusReponse.status) >= 0 && (
```

Comment:
Sorry my javascript is rusty, but could this be: ['3', '4'].includes(checkStatusResponse.status)? Or is that not a thing on vanilla arrays?

Reply:
I agree with Stefan with using `[].includes(value)`.

Use TS enum instead of harcoding `"3"` and `"4"`. The enum can be added to `/lib/type.ts`.
https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

Here's the enum of status code that match the API:

```ts
export enum StatusCode {
    APPLICATION_NO_LONGER_MEETS_CRITERIA = "99".
    FILE_BEING_PROCESSED = "1",
    NOT_ACCEPTABLE_FOR_PROCESSING = "5",
    PASSPORT_ISSUED_READY_FOR_PICKUP = "2",
    PASSPORT_ISSUED_SHIPPING_CANADA_POST = "3",
    PASSPORT_ISSUED_SHIPPING_FEDEX = "4"
}
```

Remove the comment when It's done.